presentation.go

```diff
@@ -41,7 +43,7 @@ func handleMockService(w http.ResponseWriter, r *http.Request) {
 
 func handleMockServiceList(w http.ResponseWriter, r *http.Request) {
 	setHeaders(w)
-	response := ServiceList{Services: make([]string, 0, 1)}
+	response := CCC.ServiceList{Services: make([]string, 0, 1)}
```

Comment:
If you intended to just give an empty list as parameter, use `nil`. This `make` allocates a slice that has zero elements but capacity 1, meaning one item could be added to it.

Reply:
👍